evap/staff/templates/staff_user_import.html

```diff
@@ -12,48 +12,69 @@
     {{ block.super }}
     <h3>{% trans 'Import users' %}</h3>
 
-    {% include 'staff_message_rendering_template.html' with importer_log=importer_log %}
+    {% include 'staff_user_importer.html' with importer_log=importer_log decision_form=decision_form %}
 
     <form id="user-import-form" method="POST" enctype="multipart/form-data" class="form-horizontal">
+
         {% csrf_token %}
         <div class="card mb-3">
             <div class="card-body">
                 <p>
                     {% trans 'Upload Excel file' %}
-                    (<a href="{% url 'staff:download_sample_file' 'sample_user.xlsx' %}">{% trans 'Download sample file' %}</a>,
-                    <button type="button" class="btn btn-link" onclick="copyHeaders(['Title', 'First name', 'Last name', 'Email'])">{% trans 'Copy headers to clipboard' %}</button>).
+                    (<a
+                    href="{% url 'staff:download_sample_file' 'sample_user.xlsx' %}">{% trans 'Download sample file' %}</a>,
+                    <button type="button" class="btn btn-link"
+                            onclick="copyHeaders(['Title', 'First name', 'Last name', 'Email'])">{% trans 'Copy headers to clipboard' %}</button>
+                    ).
                     {% trans 'This will create all contained users.' %}
                 </p>
                 {% include 'bootstrap_form.html' with form=excel_form %}
             </div>
         </div>
         <div class="card card-submit-area{% if test_passed %} card-submit-area-2{% endif %} text-center mb-3">
             <div class="card-body">
-                {% if not test_passed %}
-                    <button name="operation" value="test" type="submit" class="btn btn-light form-submit-btn">{% trans 'Upload and test' %}</button>
-                {% else %}
-                    <button name="operation" value="test" type="submit" class="btn btn-light form-submit-btn">{% trans 'Upload and test' %}</button>
-                    <button name="operation" value="import" type="button" onclick="importUserModalShow('import');" class="btn btn-primary form-submit-btn">{% trans 'Import previously uploaded file' %}</button>
+                <button name="operation" value="test" type="submit"
+                        class="btn btn-light form-submit-btn">{% trans 'Upload and test' %}</button>
+                {% if test_passed %}
+                    {% if not importer_log.has_decisions %}
```

Comment:
If I opt out of a "autofix" in the xlsx file, the name conflict will remain in the file, so I will always get the decision again in future "test" runs, right? This means that this is not a valid UI condition here, right?

We'd probably have to have some javascript that dynamically tests whether any decision/autofix is set to "please fix my xlsx file", and in that case, only enable the "download fixed file". When all autofixes are set to "no, the file is correct", the "import" button be enabled.